src/index.test.js

```diff
@@ -1,9 +1,9 @@
 import sinon from 'sinon'
 import configs from '../test/passing-configs'
 import failingConfigs from '../test/failing-configs'
-import { Joi } from './'
+import { validateRoot, Joi } from './'
```

Comment:
this could be `import { validateRoot as validate, Joi } from './'`

Then we could remove the `const` below.